battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketClient.java

```diff
@@ -40,7 +41,13 @@ public void send(String allShips) throws IOException {
         enemyShips = in.readLine();
     }
 
+    @Override
     public String getEnemyShips() {
```

Comment:
javadoc

Reply:
there is separate task for adding javadocs

Reply:
ok